packages/graph/hash_graph/tests/integration/postgres/links.rs

```diff
@@ -97,21 +101,27 @@ async fn get_entity_links() {
         1,
     );
 
-    let person_a_identifier = api
+    let person_a_metadata = api
         .create_entity(person_a, person_type_id.clone(), None)
         .await
         .expect("could not create entity");
```

Comment:
I'd probably avoid all of these changes and just add
```suggestion
        .expect("could not create entity")
        .identifier();
```

Reply:
I had that originally, but this caused a `temporary value dropped while borrowed` compilation error.
<img width="852" alt="image" src="https://user-images.githubusercontent.com/42802102/195827056-b4a30bd4-f301-4f15-b109-abf9b354f2b2.png">

I verified with @TimDiekmann and @thehabbos007 earlier today that what I've done here is a good workaround, but if there's a better way of handling it then let me know.

Reply:
Ah got it, it provides a reference. Bit annoying but fine as is, thanks for explaining.